### PR TITLE
Update assetlist.json

### PR DESCRIPTION
**Pull request by josietyleung**

### CODE REVIEW

File: _IBC/composablepolkadot-picasso.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "connection_id": "connection-23"
+    },
+    "channels": [
+      {
+        "chain_1": {
+          "channel_id": "channel-15",
+          "port_id": "transfer"
+        },
+        "chain_2": {
+          "channel_id": "channel-15",
```
Comment (JeremyParish69) [inline _IBC/composablepolkadot-picasso.json:20]: it seems very unlikely that both sides have the same client-id, connection-id, and channel-id; usually these are all different between chain 1 and chain 2

File: _IBC/composableparachain-picasso.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "picasso",
+      "client_id": "10-grandpa-28",
+      "connection_id": "connection-23"
+    },
+    "chain_2": {
+      "chain_name": "composable",
```
[the remark this answers is not in the document]
Comment (josietyleung) [reply _IBC/composableparachain-picasso.json]: hi they are matching + ordered

File: _IBC/composableparachain-picasso.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      "client_id": "10-grandpa-28",
+      "connection_id": "connection-23"
+    },
+    "channels": [
+      {
+        "chain_1": {
+          "channel_id": "channel-15",
```
[the remark this answers is not in the document]
Comment (josietyleung) [reply _IBC/composableparachain-picasso.json]: hi the above file is an outdated version 😊 but yes they are the exact same as it's the setup for 2 dotsama chains's ibc connections

File: composable/assetlist.json [hunk 4]
```diff
@@ -56,6 +56,41 @@
...
+      "denom_units": [
+        {
+          "denom": "ibc/3CC19CEC7E5A3E90E78A5A9ECC5A0E2F8F826A375CF1E096F4515CF09DA3E366",
+          "exponent": 0,
+          "aliases": [
+            "79228162514264337593543950342"
+          ]
+        },
+        {
+          "denom": "ksm",
```
[the remark this answers is not in the document]
Comment (josietyleung) [reply composable/assetlist.json]: updated thanks for flagging

File: _IBC/composablepolkadot-picasso.json (hunk 1 above)
Comment (josietyleung) [reply _IBC/composablepolkadot-picasso.json:20]: checked with our bridging dev. He mentioned that it was symmetric on purpose as that's what they started with connecting to two parachains with IBC. And didn't want to change it afterwards given it works

Comment (JeremyParish69) [reply _IBC/composablepolkadot-picasso.json:20]: Thanks for confirming

**Pull request merged**